bundle/store.go

```diff
@@ -6,41 +6,52 @@ package bundle
 
 import (
 	"context"
+	"encoding/base64"
+	"encoding/json"
 	"fmt"
 	"strings"
 
-	"github.com/open-policy-agent/opa/metrics"
-
 	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/metrics"
+	"github.com/open-policy-agent/opa/resolver/wasm"
```

Comment:
The bundle package was originally meant to be a simple, reusable package for interacting with bundles--over time the package has taken on dependencies with the OPA in-memory store and now we're adding a dependency on the wasm runtime. I'm a bit worried about the scope creep here. It means that anyone who wants to simply parse bundles to inspect the files/manifests will need to pull in the wasm runtime. If there's some way we can avoid these dependencies, I think it would be better.

I'd also like to see the bundle activation logic moved into a separate package because that's really an implementation detail of how we do file loading in OPA but we can solve that another time.

Reply:
I'll look into refactoring some of that, IIRC some of this was just out of convenience.

Reply:
Latest changes move the helper to instantiates the wasm resolvers into an internal bundle utils package.